src/twisted/web/server.py

```diff
@@ -30,17 +30,23 @@
 from twisted.spread.pb import Copyable, ViewPoint
 from twisted.internet import address, interfaces
 from twisted.internet.error import AlreadyCalled, AlreadyCancelled
+from twisted.protocols.policies import ProtocolWrapper, WrappingFactory
 from twisted.web import iweb, http, util
 from twisted.web.http import unquote
 from twisted.python import reflect, failure, components
 from twisted import copyright
 from twisted.web import resource
 from twisted.web.error import UnsupportedMethod
-
 from incremental import Version
 from twisted.python.deprecate import deprecatedModuleAttribute
 from twisted.logger import Logger
 
+try:
+    from twisted.protocols.tls import TLSMemoryBIOFactory
+except ImportError:
+    TLSMemoryBIOFactory = object
```

Comment:
mypy complains about this . but I have no idea how to make mypy happy while still having a code that is easy to read and use.